Sources/Sentry/SentryUIViewControllerPerformanceTracker.m

```diff
@@ -147,11 +147,14 @@ - (void)createTimeToDisplay:(UIViewController *)controller
                                            waitForFullDisplay:self.enableWaitForFullDisplay
                                          dispatchQueueWrapper:_dispatchQueueWrapper];
 
-    objc_setAssociatedObject(controller, &SENTRY_UI_PERFORMANCE_TRACKER_TTD_TRACKER, ttdTracker,
-        OBJC_ASSOCIATION_ASSIGN);
-    [ttdTracker startForTracer:(SentryTracer *)vcSpan];
+    if ([ttdTracker startForTracer:(SentryTracer *)vcSpan]) {
```

Comment:
l: Wondering if we should check if the FrameTracker is active  is this function  and don't even create a new displayTracker instance.

Reply:
I also considered this approach, but the downside is that we would have the logic of the frames tracker now in here, instead of encapsulated in the `SentryTimeToDisplayTracker`. Furthermore, the `SentryUIViewControllerPerformanceTracker` doesn't use the frames tracker and I would like to keep it that way.